js/chrome-tabs.js

```diff
@@ -24,7 +31,7 @@
 
     init(el, options) {
       this.el = el
-      this.options = options
+      this.options = extend({}, defaultOptions, options)
```

Comment:
Since `defaultOptions` has no nested options, I believe this can just be `Object.assign({}, defaultOptions, options)`.